app/assets/stylesheets/api/carts.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the api/carts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
does this file need to be here? if not, we might as well delete it since `ApplicationController` is handling all of the frontend anyway.

Reply:
Nope. There were a lot of files automatically generated. I can delete them.

Reply:
yeah we don't need this. It's just autogenerated stuff when creating a new controller